assets/html/error-pages/service-error-std.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <title>
+            Sorry, there's a problem with the service - Office for National Statistics
+        </title>
+
+        <meta charset="utf-8">
+        <meta
+            content="width=device-width,initial-scale=1.0,user-scalable=1"
+            name="viewport"
+        >
+        <meta
+            name="format-detection"
+            content="telephone=no"
+        >
+        <meta
+            name="theme-color"
+            content="#58595B"
+        >
+        <meta
+            name="apple-mobile-web-app-status-bar-style"
+            content="#58595B"
+        >
+
+        <link
+            rel="stylesheet"
+            href="https://cdn.ons.gov.uk/dp-design-system/a19df14/css/main.css"
+        >
+        <link
+            rel="shortcut icon"
+            href="https://cdn.ons.gov.uk/assets/images/favicon.ico"
+            type="image/x-icon"
+        >
+    </head>
+
+    <body>
+        <div class="ons-page">
+            <div class="ons-page__content">
+                <header class="ons-u-bb">
+                    <div class="wrapper">
+                        <div class="header col-wrap">
+                            <div class="col col--lg-one-third col--md-one-third">
+                                <a
+                                    id="logo-link"
+                                    href="/"
+                                >
+                                    <img
+                                        class="logo"
+                                        src="https://cdn.ons.gov.uk/assets/images/ons-logo/v2/ons-logo.svg"
```

Comment:
```suggestion
                                        src="/assets/images/ons-logo/v2/ons-logo.svg"
```
not a big one, but I'm surprised the host is hardcoded